uwubot.py

```diff
@@ -29,6 +33,35 @@
     # don't uwu uwubot's own messages - respond with "nyo u"
     await ctx.send(uwulib.uwu("nothing to uwu!"))
 
+# Bot Event Responses
+
+@bot.event
+async def on_ready():
+    print(uwulib.uwu(f'{bot.user.name} has connected to Discord!'))
+
+@bot.event
+async def on_message(message):
+    if 'linux' in message.content:
+        await message.channel.send(combative_linux_guy)
+    await bot.process_commands(message)
+
+# Internal Functions
+    
+combative_linux_guy = 'I\'d just like to interject for a moment. What you\'re referring to as Linux, is in fact, GNU/Linux, ' \
```

Comment:
in this totally serious project of ours, I would totally seriously recommend pulling these into text files

Reply:
legit. i was thinking the same thing. i'll take a look at what python can does.